server/services/yeelight/api/yeelight.controller.js

```diff
@@ -7,8 +7,12 @@ module.exports = function YeelightController(yeelightHandler) {
    * @apiGroup Yeelight
    */
   async function discover(req, res) {
-    const devices = await yeelightHandler.discover();
-    res.json(devices);
+    if (!this.discoveryInProgress) {
```

Comment:
You can't use `this.discoveryInProgress`, because it only exists in the `YeelightHandler`, not in the `YeelightController`.

Reply:
Oh yes ... Stupid to use it here :D